front/app/components/admin/GraphWrappers/index.tsx

```diff
@@ -172,7 +172,7 @@ export const GraphCardTitle = styled.h3`
 
 export const GraphCardFigureContainer = styled.div`
   margin-left: 10px;
-  display: inline-flex;
+  display: flex;
```

Comment:
Changed this back because I was afraid it would affect other graphs... instead of changing this I removed our dependency on this in the new card (see above)